_layouts/default.html

```diff
@@ -18,6 +18,9 @@
             iframe.width = (newwidth) + "px";
         }
     </script>
+    {% if site.google_analytics and jekyll.environment == 'production' %}
+    {% include analytics.html %}
+    {% endif %}
```

Comment:
Ok, so if this is working right, I should see the snippet from `analytics.html` appear here in the `<head>` of each page?

Reply:
Yes